articles/sql-database/sql-database-connectivity-architecture.md

```diff
@@ -45,14 +45,17 @@ If you are connecting from outside Azure, your connections have a connection pol
 ![architecture overview](./media/sql-database-connectivity-architecture/connectivity-from-outside-azure.png)
 
 > [!IMPORTANT]
-> When using service endpoints with Azure SQL Database your policy is **Redirect** by default. So to enable connectivity from inside your Vnet you must allow outbound to all Azure SQL Database IP addresses, not just the gateway IPs. This can be done with the help of NSG (Network Security Group) Service Tags, if you want to allow outbound only to gateway IPs please change your setting to **Proxy**.
+> When using service endpoints with Azure SQL Database your policy is **Proxy** by default. So to enable connectivity from inside your Vnet you must allow outbound to Azure SQL Database Gateway IP addresses specified in the list below.
 
 ## Azure SQL Database gateway IP addresses
 
 To connect to an Azure SQL database from on-premises resources, you need to allow outbound network traffic to the Azure SQL Database gateway for your Azure region. Your connections only go via the gateway when connecting in Proxy mode, which is the default when connecting from on-premises resources.
 
 The following table lists the primary and secondary IPs of the Azure SQL Database gateway for all data regions. For some regions, there are two IP addresses. In these regions, the primary IP address is the current IP address of the gateway and the second IP address is a failover IP address. The failover address is the address to which we might move your server to keep the service availability high. For these regions, we recommend that you allow outbound to both the IP addresses. The second IP address is owned by Microsoft and does not listen in on any services until it is activated by Azure SQL Database to accept connections.
 
+> [!IMPORTANT]
+> If you are connecting from within Azure your connection policy will be **Redirect** by default (except if you are using service endpoints). It will not be sufficient to allow the following IPs you must allow all Azure SQLDB IPs. If you are connecting from within a Vnet this can be accomplished with the help of NSG (Network Security Groups) Service Tags, for more information see [Service Tags](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview#service-tags).
```

Comment:
Change to "If you are connecting from within Azure, your connection policy is **Redirect** by default (except if you are using service endpoints). It is not sufficient to allow the following IPs - you must also allow all Azure SQLDB IPs. If you are connecting from within a Vnet, this can be accomplished with the help of NSG (Network Security Groups) Service Tags. For more information, see [Service Tags](https://docs.microsoft.com/en-us/azure/virtual-network/security-overview#service-tags)."